interpreter/cling/lib/Interpreter/ASTImportSource.cpp

```diff
@@ -132,21 +162,24 @@ namespace cling {
       } else {
         // Get the identifier info from the parent interpreter
         // for this Name.
-        llvm::StringRef name(childDeclName.getAsString());
-        IdentifierTable &parentIdentifierTable =
-          m_parent_Interp->getCI()->getASTContext().Idents;
-        IdentifierInfo &parentIdentifierInfo = parentIdentifierTable.get(name);
-        DeclarationName parentDeclNameTemp(&parentIdentifierInfo);
-        parentDeclName = parentDeclNameTemp;
+        std::string name = childDeclName.getAsString();
+        if (!name.empty()) {
+          llvm::StringRef nameRef(name);
```

Comment:
No need to build an explicit `StringRef`; passing `name` to `get()` below will construct it in place.